stable/coredns/values.yaml

```diff
@@ -63,6 +63,7 @@ service:
 # loadBalancerIP: ""
 # externalIPs: []
 # externalTrafficPolicy: ""
+  name:
```

Comment:
This should be commented as we by default want to keep original behavior.

Reply:
This should NOT be commented out, but you do want the value to be an empty string `""` so `"coredns.fullname"` is used by default/keeps the original behavior.

Reply:
This does keep the original behavior if no value is given, similar to how the `serviceAccount.name` behaviors at https://github.com/coredns/helm/blob/master/stable/coredns/values.yaml#L72 and https://github.com/coredns/helm/blob/master/stable/coredns/templates/_helpers.tpl#L145

Or did I misundestand something?

Reply:
No, you’re good @livyfu but I recommend using an empty string for clarity and type instead of no value at all which YAML treats as null.

Reply:
Point taken, and thank you!